extractor/build.gradle

```diff
@@ -25,6 +25,28 @@ application {
 
 tasks.withType(JavaCompile) {
     options.release = 21
+
+    // Specify the directory for generated source files
+    options.compilerArgs << "-s"
+    options.compilerArgs << "$buildDir/generated/sources"
```

Comment:
Bad magic.
Let's not create files in OS temp file, that is always outside of repo.

Reply:
Or just create string content and do not save it to file keep as string. Load expected file to string and compare two strings.

Reply:
> Or just create string content and do not save it to file keep as string. Load expected file to string and compare two strings.

so, no saving to build? shall i revert changes made to build.gradle for build directory saving?

Reply:
Yes, let's remove, it will be more stable solution.